bin/postgres-ha/pgbackrest/pgbackrest-create-replica.sh

```diff
@@ -81,7 +82,7 @@ then
 fi
 
 # perform the restore
-pgbackrest restore "${restore_cmd_args[@]}"
-err_check "$?" "pgBackRest Replica Creation" "pgBackRest restore failed when creating replica"
+eval "pgbackrest restore ${restore_cmd_args[*]} ${RESTORE_OPTS}"
```

Comment:
Does `RESTORE_OPTS` contain references to other environment variables? If not, this `eval` can be dropped.
```suggestion
pgbackrest restore "${restore_cmd_args[@]}" ${RESTORE_OPTS}
```

Reply:
I moved to `eval` because otherwise pgBackRest was unhappy with the quotes potentially included in the restore opts when running the pgBackRest restore command, e.g `--target='2020-07-01 20:23:52.05903+00'` or `--target="2020-07-01 20:23:52.05903+00"`.

By switching to `eval` all the quotes are handled properly and pgBackRest is happy (without having to deal with additional escaping, etc.).